playground/p/issues/4672/_import.info.json

```diff
@@ -0,0 +1,11 @@
+{
+  "source": "https://lit.dev/playground/#gist=aece301df423847e72b1a85e1e9caf75",
```

Comment:
Is this intended to be committed? If so, it seems weird to name it by issue.

Reply:
Yes. In working on Virtualizer issues, I've found that committing a minimal repro, followed by the fix, makes for a very convenient PR review process. A reviewer can check out the PR branch, temporarily detach HEAD to view the repro without the fix, then switch back to the tip of the branch to review the fix.

I've also found through experience that when a repro case is part of a workflow like this, it's easiest to use the issue number as the name of the repro, to minimize cognitive load.